consensus/src/chained_bft/block_storage/block_store_test.rs

```diff
@@ -255,6 +257,7 @@ fn test_insert_vote() {
     let block_store = inserter.block_store();
     let genesis = block_store.root();
     let block = inserter.insert_block_with_qc(certificate_for_genesis(), &genesis, 1);
+    let mut pending_votes = PendingVotes::new();
```

Comment:
this test can be moved to pending_vote_test

Reply:
Decided to keep it at `block_store_tests` to test that a block can be successfully added after a qc is formed.

Reply:
inserting block after a qc is probably tested somewhere else in block_store_test and probably should be separated from aggregating a qc

Reply:
Looks like this test is the only place insert block after qc is tested?